p2p/simulations/network.go

```diff
@@ -82,6 +82,64 @@ func (net *Network) Events() *event.Feed {
 	return &net.events
 }
 
+func (net *Network) AddNodes(nodes int) ([]enode.ID, error) {
```

Comment:
How does this relate to for example `NewNodeWithConfig`? Will it be confusing that they have rather different names but both have to do with adding nodes?

Reply:
`NewNodesWithRandomConfig`?